powerfactory_tools/interface.py

```diff
@@ -166,7 +176,7 @@ def load_powerfactory_module_from_path(self) -> PFTypes.PowerFactoryModule:
         spec.loader.exec_module(pfm)
         return t.cast("PFTypes.PowerFactoryModule", pfm)
 
-    def load_settings_dir_from_pf(self) -> PFTypes.DataDir:
+    def load_settings_dir_from_pf(self) -> PFTypes.DataObject:
```

Comment:
whats the reasoning here?

Reply:
the external data container assigned in the project settings is assumed as type DataDir

Therefore, the directories for storing the settings (units, fonts, etc.) should not be from the type DataDir as they are from different classes in PowerFactory. Either one have to use the basis class DataObject or one have to specify speparate directory types.

Reply:
`DataDir` is a superclass for data dirs which at the moment does not differ from a `DataObject`. However, 
it is used here to indicate that its not a generic `DataObject` and may gain some additional attributes in the future.